client/perl-lib/SMT/Agent/Constants.pm

```diff
@@ -35,18 +35,14 @@ use constant LOG_FILE		=> '/var/log/smtclient.log';
 use constant CLIENT_CONFIG_FILE	=> '/etc/sysconfig/smt-client';
 
 # smt client credenitials file containing guid
-#  SLE11 => /etc/zypp/credentials.d/NCCcredentials
+#  SLE11 => /etc/zypp/credentials.d/SCCcredentials
 #  SLE12 => /etc/zypp/credentials.d/SCCcredentials
 # SLE12 file may be missing due to update or missing SUSEConnect package
 #  in that case fallback to SLE11 file only if SLE11 file is present:
 sub CREDENTIALS_FILE() {
   my $cred_path = '/etc/zypp/credentials.d';
-  my $cred_file_sle11 = 'NCCcredentials';
   my $cred_file_sle12 = 'SCCcredentials';
   my $cred_file = $cred_file_sle12;
-  if ( ! -e "$cred_path/$cred_file_sle12"  && -e "$cred_path/$cred_file_sle11"  ) {
-    $cred_file = $cred_file_sle11;
-  }
   return "$cred_path/$cred_file";
 }
 
```

Comment:
Make sense. yast module does the same. I think I wasn't sure if they are compatible.